src/EnrollWebauthnController.js

```diff
@@ -113,8 +114,14 @@ function (Okta, Errors, FormType, FormController, CryptoUtil, FidoUtil, webauthn
     },
 
     Form: {
-      title: _.partial(Okta.loc, 'enroll.u2f.title', 'login'),
-      save: _.partial(Okta.loc, 'enroll.u2f.save', 'login'),
+      title: function () {
+        return webauthn.isNewApiAvailable() ?
```

Comment:
do we need this check? This file is specifically for `webauthn`. Before this change `webauthn` factor was showing the labels for `u2f`, but after this change isn't the intention that `webauthn` will show only `webauthn` labels? Do we still need a backup of `webauthn` working exactly as u2f?

Reply:
I had this question too and I spoke with @mpeng-okta about this. The backend still defaults to u2f as well, so I left the front-end to do the same